src/main/java/service/strategy/strategyimpl/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package service.strategy.strategyimpl;
+
+import dao.Storage;
+import model.FruitRecord;
+
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
Don't begin class or method implementation with empty line. Please remove the redundant empty lines before the class declaration.